src/fides/api/models/privacy_request.py

```diff
@@ -1209,17 +1208,14 @@ def save_filtered_access_results(
         if not self.policy.get_rules_for_action(action_type=ActionType.access):
             return None
 
-        self.filtered_final_upload = json.dumps(results, cls=CustomJSONEncoder)
+        self.filtered_final_upload = results
         self.save(db)
 
         return None
 
     def get_filtered_access_results(self) -> Dict[str, Dict[str, List[Row]]]:
         """Fetched the same filtered access results we uploaded to the user"""
-        return json.loads(
-            self.filtered_final_upload or "{}",
-            object_hook=_custom_decoder,
-        )
+        return self.filtered_final_upload or {}
```

Comment:
Hm same here, this might be renamed since it's doing less than it was previously